sklearn/metrics/tests/test_classification.py

```diff
@@ -1193,6 +1091,12 @@ def test__check_targets():
                 assert_array_equal(y2out, np.squeeze(y2))
             assert_raises(ValueError, _check_targets, y1[:-1], y2)
 
+    # Make sure seq of seq is not supported
+    y1 = [(1, 2,), (0, 2, 3)]
+    y2 = [(2,), (0, 2,)]
+    assert_raise_message(ValueError, "unknown is not supported",
```

Comment:
@arjoly I'm now wondering if this is too cruel for users who missed the memo. Should we instead be repurposing `is_sequence_of_sequences` to `raise_if_sequence_of_sequences`...?